mypyc/test-data/irbuild-isinstance.test

```diff
@@ -0,0 +1,97 @@
+[case testIsinstanceInt]
+def is_int(value):
+    return isinstance(value, int)
```

Comment:
The IR could be a little simpler if the function would have an annotations, e.g. `def is_int(value: object) -> bool`. IF there is no annotation, more things have `Any` types, often resulting in some needless boxing.

Not a big deal though.